core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -2227,9 +2227,10 @@ public List<AlluxioURI> getInAlluxioFiles() throws UnavailableException {
       // Root should always exist.
       throw new RuntimeException(e);
     }
+    int fileCount = Configuration.getInt(PropertyKey.MASTER_WEB_IN_ALLUXIO_DATA_PAGE_COUNT);
 
     try (LockedInodePath inodePath = rootPath) {
-      getInAlluxioFilesInternal(inodePath, files);
+      getInAlluxioFilesInternal(inodePath, files, fileCount);
```

Comment:
nit: can inline `Configuration.getInt` since the var isn't used anywhere else

Reply:
done